### ISSUE TITLE
i am not able to run the youtube downloader program.It is showing error

### ISSUE THREAD

**Issue by sunkarisukesh** [tuomaskivioja/3-Simple-Python-Projects#2]
![Screenshot (264)](https://user-images.githubusercontent.com/95559724/194706132-f2c6219c-3dae-4fb4-b971-d3f41ac051d3.png)

**Comment by RubenMont90** [comment]
> ![Screenshot (264)](https://user-images.githubusercontent.com/95559724/194706132-f2c6219c-3dae-4fb4-b971-d3f41ac051d3.png)
not sure if still relevant but I've encountered the same issue, and managed to fix it by adding an extra backslash, in your case would work with this path: `"C:\\Users\\sukesh\\Desktop"`.

**Comment by ghost** [comment]
you can also add 'r' right before the path string

**Comment by azeemprogrammer** [comment]
Yes, I have modified this old script, and it works Perfectly. As for why the old code was not working, the error you encountered, `IndexError: list i`ndex out of range, was because you were trying to access `argv[1]` to get the YouTube URL from the command-line arguments

**Comment by Nandini1071** [comment]
is this issue still open ?

**Comment by dayajiji** [comment]
I solved this problem by putting (0) instead of (1) and also inverting the slash / and it worked 
![Captura de tela 2024-04-18 173707](https://github.com/tuomaskivioja/3-Simple-Python-Projects/assets/136419156/95b606ca-57af-479e-8db5-d3958fdde94b)

**Comment by dayajiji** [comment]
I solved this problem by putting (0) instead of (1) and also inverting the slash and it worked
![Captura de tela 2024-04-18 173707](https://github.com/tuomaskivioja/3-Simple-Python-Projects/assets/136419156/a6f3a498-dfee-466b-afcd-042c36af67ad)
